recipe/bld.bat

```diff
@@ -1,4 +1,23 @@
 robocopy . "%PREFIX%" /S /XF bld.bat
 if errorlevel NEQ 1 exit 1
 
+cd ..
```

Comment:
Is it ok to use the parent directory or should we create a subfolder as a working directory for `get_plugins.py`?

Reply:
TL;DR Yes, it should be.

This generally puts it in the `work` directory of `conda-build`, which is fair game for build scratch space. With `conda-build` 2+ the `work` directory actually sits within a temporary directory that `conda-build` creates for each build (this remains unique even if the same recipe is built multiple times due to a random number tacked on). IOW this scratch space is independent of any other build's scratch space. That whole temporary directory gets purged at the end. This stuff (belonging to that directory) would be removed as part of that purge. So this should be fine.